js/src/personal/maliha/About.tsx

```diff
@@ -0,0 +1,144 @@
+import styles from './Maliha.module.css'
+
+export function About() {
+  return (
+    <>
+      <Anchor
+        href="https://youtu.be/VgoE837TskI?si=o8p281ughP0aUxmM"
+        target="_blank"
+      >
+        <Text
+          inherit
+          component="span"
+          className={styles.title}
+          id={styles.title2}
+          variant="gradient"
+          gradient={{
+            from: '#dc1d1d',
+            to: 'rgb(151,92,164)',
+            deg: 50,
+          }}
+          pos="absolute"
+          pl={1440}
+        >
+          {'about me'}
+        </Text>
+      </Anchor>
+      <Text
+        inherit
+        component="span"
+        className={styles.title}
+        id={styles.title3}
+        variant="gradient"
+        gradient={{
+          from: '#dc1d1d',
+          to: 'rgb(151,92,164)',
+          deg: 50,
+        }}
+        pos="absolute"
+        pl={2000}
+      >
+        {'cs & math @ hunter | class of 2026'}
+      </Text>
+      <Text
+        inherit
+        component="span"
+        className={styles.title}
+        id={styles.title3}
+        variant="gradient"
+        gradient={{
+          from: '#dc1d1d',
+          to: 'rgb(151,92,164)',
+          deg: 50,
+        }}
+        pos="absolute"
+        pl={2570}
+        top={260}
+      >
+        {'i have birds'}
+      </Text>
+      <Text
+        inherit
+        component="span"
+        className={styles.title}
+        id={styles.title3}
+        variant="gradient"
+        gradient={{
+          from: '#dc1d1d',
+          to: 'rgb(151,92,164)',
+          deg: 50,
+        }}
+        pos="absolute"
+        pl={2005}
+        top={450}
+      >
+        {'i draw (rarely)'}
+      </Text>
+      <Text
+        inherit
+        component="span"
+        className={styles.title}
+        id={styles.title3}
+        variant="gradient"
+        gradient={{
+          from: '#dc1d1d',
+          to: 'rgb(151,92,164)',
+          deg: 50,
+        }}
+        pos="absolute"
+        pl={2900}
+        top={620}
+      >
+        {'i bike in central park'}
+      </Text>
+      <Text
+        inherit
+        component="span"
+        className={styles.title}
+        id={styles.title3}
+        variant="gradient"
+        gradient={{
+          from: '#dc1d1d',
+          to: 'rgb(151,92,164)',
+          deg: 50,
+        }}
+        pos="absolute"
+        pl={2300}
+        top={430}
+      >
+        {'currently watching: '}
+        <br />
+        {'bridgerton'}
+        <br />
+        {'the boys'}
+        <br />
+        {'queen charlotte'}
+      </Text>
+      <Text
+        inherit
+        component="span"
+        className={styles.title}
+        id={styles.title3}
+        variant="gradient"
+        gradient={{
+          from: '#dc1d1d',
+          to: 'rgb(151,92,164)',
+          deg: 50,
+        }}
+        pos="absolute"
+        pl={2600}
+        top={380}
+      >
+        <br />
```

Comment:
nit: using proper list html tags will make it easier to read parse for screen readers and other accessibility use cases. Not required for personal website ofc